### ISSUE TITLE
Include types

### ISSUE THREAD

**Issue by NoxHarmonium** [adrianjost/dedent-tabs#208]
If you are using this library with Typescript you need to create a definition file with the following in it, otherwise the compiler will complain:

```
// Derived from https://github.com/DefinitelyTyped/DefinitelyTyped/blob/f7ec78508c6797e42f87a4390735bc2c650a1bfd/types/dedent/index.d.ts
declare module "dedent-tabs" {
  export = dedent;

  function dedent(literals: string): string;
  function dedent(
    literals: TemplateStringsArray,
    ...placeholders: readonly unknown[]
  ): string;
  namespace dedent {}
}
```

It should be pretty straightforward to include these in the NPM package, I don't think it has to be done through Definitely Typed.

I think the steps would be:
1. Include a dedent-tabs.d.ts file with the above content in it in the published package
2. Add a "types" entry to package.json (e.g. `"types": "dist/dedent-tabs.d.ts",`)

I'll see if I can look at getting a PR up for that. Thanks!

**Comment by adrianjost** [comment]
Hi @NoxHarmonium thank you for taking the time to write this detailed issue. I agree that we should provide correct TS definitions.
I will add it to my list of open tasks but any PR that fixes this faster than me would as always be appreciated.

**Comment by adrianjost** [comment]
released in v0.10.2
